docs/jmeter/writing-tests.md

```diff
@@ -396,6 +397,8 @@ Some tests require files as images, JAR files, etc. You can provide this from a
 
 The data will be cloned from the bucket to the volume using [Rclone](https://rclone.org/) and will be available to all the pods.
 
-For the full list of possible environment variables check [Kangal environment variables](env-vars.md)
+For the full list of possible environment variables check [Kangal environment variables](../env-vars.md)
```

Comment:
I believe for TechDocs is better to have the fullpath, no?

Reply:
It's ok when addressing to the files in the same /docs folder. I testes it locally and worked fine.